framework/src/modules/http_api/components/storage/entities/block.js

```diff
@@ -18,6 +18,7 @@ const path = require('path');
 const assert = require('assert');
 const {
 	entities: { Block: BlockEntity },
+	// eslint-disable-next-line
 } = require('../../../../../components/storage');
 
 const sqlFiles = {
```

Comment:
sql file?

Reply:
this is http_api, i just kept it until #5213

Reply:
Don't get it. Since all migrations are already removed, storage object are removed. Even the Jenkins don't start the database then why to keep these migrations?